src/c++/perf_analyzer/test_load_manager_base.h

```diff
@@ -106,6 +106,48 @@ class TestLoadManagerBase {
       }
     }
   }
+  // FIXME: remove the print statements
+  //
+  void CheckSharedMemory()
+  {
+    auto stats = GetStats();
+
+    if (params_.shared_memory_type == SYSTEM_SHARED_MEMORY) {
```

Comment:
For values that always remain 0, should they be either excluded from this check and/or should their be new tests added to cause their values to increase from 0?

Reply:
Discussed offline